types/nodemailer-ses-transport/index.d.ts

```diff
@@ -3,24 +3,23 @@
 // Definitions by: Seth Westphal <https://github.com/westy92/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare module "nodemailer-ses-transport" {
-	import * as AWS from 'aws-sdk';
-	import * as nodemailer from "nodemailer";
+import * as AWS from 'aws-sdk';
```

Comment:
Should we make quote usage consistent?  (`"` vs. `'`)
https://palantir.github.io/tslint/rules/quotemark/